src/interactive/tooltip.test.jsx

```diff
@@ -40,14 +100,33 @@ describe('Tooltip', () => {
 		const rightDropdownWrapper = mount(rightDropdown);
 		const trigger = rightDropdownWrapper.find('.dropdown-trigger');
 
-		it('applies correct alignment className to dropdown content', () => {
+		it('applies correct alignment className to tooltip content', () => {
 			trigger.simulate('focus');
 			const content = rightDropdownWrapper.find('.dropdown-content');
 
 			expect(content.prop('className')).toContain('dropdown-content--right');
 		});
 	});
 
+	describe('right aligned tooltip', () => {
+		const aboveDropdown = (
+			<Tooltip
+				popAbove
+				trigger={dropdownTrigger}
+				content={dropdownContent}
+			/>
+		);
+		const aboveDropdownWrapper = mount(aboveDropdown);
+		const trigger = aboveDropdownWrapper.find('.dropdown-trigger');
+
+		it('applies correct alignment className to tooltip content', () => {
+			trigger.simulate('focus');
```

Comment:
For this, it doesn't look like you need to trigger `focus` - the markup appears to be rendered either way. If that's the case, you could just cover this (and more) with a snapshot test. The test above for `dropdown-content` would also be redundant

Reply:
Removing second test and adding a snapshot test 👍